configurations/sync_imaqal_pool_to_text_it.py

```diff
@@ -0,0 +1,86 @@
+from core_data_modules.cleaners import Codes, somali
+from dateutil.parser import isoparse
+from src.pipeline_configuration_spec import *
+
+
+PIPELINE_CONFIGURATION = PipelineConfiguration(
+    pipeline_name="IMAQAL-Demographics-To-TextIt",
```

Comment:
I think we should use RapidPro instead of TextIt for consistency with pipeline config.

Reply:
hmm we use rapid_pro in the source code to show that it is compatible with the rapid_pro software (of which Rapid Pro and TextIt are 2 of many instances). Since we a syncing to a concrete TextIt workspace here I think we can call it TextIt, just like how we refer to the credentials tokens for this workspace as being for "textit" rather than "rapid pro".